tests/ignite/contrib/metrics/regression/test_median_relative_absolute_error.py

```diff
@@ -186,7 +188,7 @@ def update(engine, i):
         e = np.abs(np_y_true - np_y_preds) / np.abs(np_y_true - np_y_true.mean())
         np_res = np.median(e)
 
-        assert pytest.approx(res) == np_res
+        assert pytest.approx(res, rel=1e-3) == np_res
```

Comment:
Why you up the rel tolerance ? If it starts failing than there can be something in the code as well.